soc/nordic/nrf54l/Kconfig.defconfig.nrf54l15_cpuapp

```diff
@@ -0,0 +1,14 @@
+# Nordic Semiconductor nRF54L15 MCU
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF54L15_CPUAPP
+
+config NUM_IRQS
+	default 271
+
+config IEEE802154_NRF5
+	default IEEE802154
```

Comment:
is this needed?

Reply:
It looks like other SoCs use this option. I would leave it as is for this PR and I will check if it's needed for this and other SoCs afterwards.

Reply:
should not be needed, option is already default y based on DT. Just remove it and other socs can be updated later.

Reply:
removed